### ISSUE TITLE
Return added addresses with their ids when calling `client.addKvRecords`

### ISSUE THREAD

**Issue by douglance** [GridPlus/gridplus-sdk#242]
If we return the added records with ids, then the consuming project can combine the added records to the existing list without needing to refetch. 

Without ids, it's not possible to use the other functions of the client on those records because they require the id.

**Comment by alex-miller-0** [comment]
Good idea. Please create a corresponding ticket in the firmware repo and link it to this one.
